pom.xml

```diff
@@ -231,7 +231,9 @@
     <!-- org.fusesource.leveldbjni will be used except on arm64 platform. -->
     <leveldbjni.group>org.fusesource.leveldbjni</leveldbjni.group>
     <kubernetes-client.version>6.13.3</kubernetes-client.version>
-    <okio.version>1.17.6</okio.version>
+    <okio.version>3.9.0</okio.version>
+    <okhttp.version>4.12.0</okhttp.version>
```

Comment:
Will this affect us?
<img width="1025" alt="image" src="https://github.com/user-attachments/assets/7eb73b8e-83d9-438b-b229-24350e240f28">
https://github.com/fabric8io/kubernetes-client/blob/main/doc/KubernetesClientWithIPv6Clusters.md

Reply:
I have uncompressed the kotlin-stdlib-2.0.10.jar file and all class files are part of the kotlin directory / package:
```
[roczei@roczei-MBP16 2.0.10]$ jar tf kotlin-stdlib-2.0.10.jar | head
META-INF/
META-INF/MANIFEST.MF
META-INF/kotlin-stdlib.kotlin_module
kotlin/
kotlin/ArrayIntrinsicsKt.class
kotlin/BuilderInference.class
kotlin/CharCodeJVMKt.class
kotlin/CharCodeKt.class
kotlin/CompareToKt.class
kotlin/ContextFunctionTypeParams.class
[roczei@roczei-MBP16 2.0.10]$ 
```

All Spark unit tests have passed in this separate pull request: roczei#4. Currently I can only validate this, I hope this will be enough.